inception/inception-api-annotation-experimental/src/main/java/de/tudarmstadt/ukp/inception/experimental/api/messages/response/relation/CreateRelationResponse.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Licensed to the Technische Universität Darmstadt under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The Technische Universität Darmstadt
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.experimental.api.messages.response.relation;
+
+import java.util.List;
+
+import de.tudarmstadt.ukp.clarin.webanno.api.annotation.model.VID;
+
+public class CreateRelationResponse
```

Comment:
Is this the message the client gets in response to a CreateRelationRequest? Does the client also get it if a relation has been created by another user or by the system - so not in "response" to a request by the client? If the latter, then "response" might be a misleading name.

Reply:
Yes, it is the latter. Initially I only thought it about a direct response, but due to the broker mechanic anyone can receive the message.
How about: *Message instead of *Response?